app/posts/register-trainee-teachers/2021-02-05-how-our-users-behave-task-list.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Reflect how providers work through the task list
+description: Our previous iteration of the task list did not reflect the way our users think and behave.
+date: 2021-02-05
+---
+
+Our previous iteration of the task list did not reflect the way our users work. Here's what we presented to users:
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "task-list-version-one.png"
+  }
+}) }}
```

Comment:
Should include some alternative text for images. This can be done 2 ways.

1. Add alt text to image (doesn’t appear in browser, but does get announced):

```
{{ appFigure({
  image: {
    file: "task-list-version-one.png",
    alt: "My great alternative text."
  }
}) }}
```

2. Add a figure caption: this will be shown below the image:

```
{{ appFigure({
  image: {
    file: "task-list-version-one.png"
  },
  caption: "My great caption text."
}) }}
```

Reply:
I have attempted both @paulrobertlloyd :)